site/src/pages/DeploymentSettingsPage/IdpOrgSyncPage/IdpOrgSyncPageView.tsx

```diff
@@ -0,0 +1,354 @@
+					<div className="flex flex-row">
+						<div className="grid items-center gap-1">
+							<Label className="text-sm" htmlFor={SYNC_FIELD_ID}>
+								Organization sync field
+							</Label>
+							<div className="flex flex-row items-center gap-5">
+								<div className="flex flex-row gap-2 w-72">
+									<Input
+										id={SYNC_FIELD_ID}
+										value={form.values.field}
+										onChange={async (event) => {
+											void form.setFieldValue("field", event.target.value);
+										}}
+									/>
+									<Button
+										className="w-20"
+										disabled={form.isSubmitting || !form.dirty}
+										onClick={(event) => {
+											event.preventDefault();
+											form.handleSubmit();
+										}}
+									>
+										Save
+									</Button>
+								</div>
+								<div className="flex flex-row items-center gap-3">
+									<Switch
+										id={ORGANIZATION_ASSIGN_DEFAULT_ID}
+										checked={form.values.organization_assign_default}
+										onCheckedChange={async (checked) => {
+											void form.setFieldValue(
+												"organization_assign_default",
+												checked,
+											);
+											form.handleSubmit();
+										}}
+									/>
+									<span className="flex flex-row items-center gap-1">
+										<Label htmlFor={ORGANIZATION_ASSIGN_DEFAULT_ID}>
+											Assign Default Organization
+										</Label>
+										<AssignDefaultOrgHelpTooltip />
+									</span>
+								</div>
+							</div>
+							<p className="text-content-secondary text-2xs m-0">
+								If empty, organization sync is deactivated
+							</p>
+						</div>
+					</div>
+
+					<div className="flex flex-col gap-4">
+						<div className="flex flex-row pt-8 gap-2 justify-between items-start">
+							<div className="grid items-center gap-1">
+								<Label className="text-sm" htmlFor={IDP_ORGANIZATION_NAME_ID}>
+									IdP organization name
+								</Label>
+								<Input
+									id={IDP_ORGANIZATION_NAME_ID}
+									value={idpOrgName}
+									className="min-w-72 w-72"
+									onChange={(event) => {
+										setIdpOrgName(event.target.value);
+									}}
+								/>
+							</div>
+							<div className="grid items-center gap-1 flex-1">
+								<Label className="text-sm" htmlFor=":r1d:">
+									Coder organization
+								</Label>
+								<MultiSelectCombobox
+									className="min-w-60 max-w-3xl"
+									value={coderOrgs}
+									onChange={setCoderOrgs}
+									defaultOptions={organizations.map((org) => ({
+										label: org.display_name,
+										value: org.id,
+									}))}
+									hidePlaceholderWhenSelected
+									placeholder="Select organization"
+									emptyIndicator={
+										<p className="text-center text-md text-content-primary">
+											All organizations selected
+										</p>
+									}
+								/>
+							</div>
+							<div className="grid items-center gap-1">
+								<div>&nbsp;</div>
```

Comment:
Does the non-breaking space need its own div? Couldn't we just put it in directly, and have it be treated as a text node?